python-hadoop/hadoop/io/BytesWritable.py

```diff
@@ -28,4 +28,4 @@ def readFields(self, data_input):
         self._value = data_input.readFully(size)
 
     def toString(self):
-        return ''.join(chr(x) for x in self._value)
+        return ''.join(chr(x) for x in self._value if x > 31 or x == 9)
```

Comment:
The mod 256 method adds some unprintable characters. Mod 128 doesn't, though. I suppose it really depends on what the goal of toString is. If our goal is to not blow up when making a best effort to stringify binary data then I agree that the mod 255 method fits the bill